catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/BqsConverter.java

```diff
@@ -784,7 +784,7 @@ public void exitQuoted_string(BqsParser.Quoted_stringContext ctx) {
             Filter filter = null;
             if (StringUtils.isNotBlank(attribute)) {
                 if (attribute.equals(MetacardVersion.ACTION)) {
-                    if (stringValue.equals(NsiliCardStatus.NEW)) {
+                    if (stringValue.equals(NsiliCardStatus.CHANGED)) {
```

Comment:
I thought we only had history on CHANGED/DELETED cards and wouldn't have a history card for NEW anymore. So if we don't have a history card, then we should mark it new? I thought thats what these changes were doing, but maybe I missed something.

For NSILI, they have NEW, CHANGED, OBSOLETE.